luarules/gadgets/unit_collision_damage_behavior.lua

```diff
@@ -94,7 +107,7 @@ end
 local function velocityDamageDirection(unitID)
```

Comment:
```suggestion
local function applyFallDamage(unitID)
```
This function returns a boolean, not a direction. What it really does is indicate whether to apply fall damage?

Reply:
Maybe "isValidCollisionDirection"

Because it handles units bumping into other units, negating damage from units being fallen upon by other units, and units falling at mostly downward angles.

Reply:
That sounds good, make it so

Reply:
done